sentry-spring-boot-starter/src/main/java/io/sentry/spring/autoconfigure/SentryAutoConfiguration.java

```diff
@@ -58,6 +54,39 @@ public ServletContextInitializer sentryServletContextInitializer() {
         return new SentryServletContextInitializer();
     }
 
+    /**
+     * Initializes a {@link List<EventBuilderHelper>}.
+     *
+     * @return a new instance of {@link List<EventBuilderHelper>}.
+     */
+    @Bean
+    @ConditionalOnMissingBean(EventBuilderHelper.class)
```

Comment:
How would this interact if the user application had similar logic to provide a different list of event builder helpers?

Reply:
This init method will be ignored, if spring context already has a list of builder helpers.

Reply:
![image](https://user-images.githubusercontent.com/5486149/72406708-209df180-3798-11ea-839f-108bf83b2af1.png)

Reply:
How does spring ensure the order of initialization? I.e. how is it ensured that these initializers are only used after the application-defined ones?

Reply:
ok, There's a lot to say about this. In short, when initing sentry client and spring context don't has this beans, spring will search initializers which providing beans.